roles/rsyslog/roles/input_roles/ovirt/defaults/main.yaml

```diff
@@ -46,7 +46,7 @@ rsyslog_conf_local_ovirt_modules:
 
       - comment: 'Log messages sent to local UNIX socket with use=off'
         options: |-
-          module(load="imuxsock" SysSock.use="off")
+          #module(load="imuxsock" SysSock.use="off")
```

Comment:
an accident?

Reply:
This setting does not work with default rsyslog conf. We are getting an error message. @nhosoi

Reply:
I see since it's already there...

https://github.com/linux-system-roles/logging/blob/master/roles/rsyslog/templates/etc/rsyslog.conf.j2#L19-L20

I wonder could you generate the file only when rsyslog_default is false?

Reply:
Thank you. I updated the state of the file instead with the condition as you suggested
Need to consider if we should move this to the rsyslog_common_rules and remove from viaq, debops and ovirt sub-roles. @nhosoi